pkg/imgpkg/bundle/bundle.go

```diff
@@ -45,32 +47,95 @@ func (o *Bundle) DigestRef() string { return o.plainImg.DigestRef() }
 func (o *Bundle) Repo() string      { return o.plainImg.Repo() }
 func (o *Bundle) Tag() string       { return o.plainImg.Tag() }
 
-func (o *Bundle) Pull(outputPath string, ui ui.UI) error {
+func (o *Bundle) Pull(outputPath string, ui goui.UI, pullNestedBundles bool) error {
+	return o.pull(outputPath, ui, pullNestedBundles, "", map[string]bool{}, 0)
+}
+
+func (o *Bundle) pull(baseOutputPath string, ui goui.UI, pullNestedBundles bool, bundlePath string, imagesProcessed map[string]bool, numSubBundles int) error {
 	img, err := o.checkedImage()
 	if err != nil {
 		return err
 	}
 
-	ui.BeginLinef("Pulling bundle '%s'\n", o.DigestRef())
+	if o.rootBundle(bundlePath) {
+		ui.BeginLinef("Pulling bundle '%s'\n", o.DigestRef())
+		ui.BeginLinef("Bundle Layers\n")
```

Comment:
is this heading only printed for the root bundle? if so, it's a little weird to print it for root bundle and not for layers of nested bundles.

Reply:
Yes, this was being only printed for the root bundle. Went ahead and dropped  printing "Bundle Layers" when a root bundle is being processed.